guides/common/modules/ref_supported-host-operating-systems-and-architectures-in-registration.adoc

```diff
@@ -0,0 +1,31 @@
+[id="supported-host-operating-systems-and-architectures-in-registration_{context}"]
+= Supported Host Operating Systems and Architectures in Registration
+
+{Project} supports the following operating systems and architectures for host registration:
+
+Supported Host Operating Systems::
+
+Hosts must use one of the following {RHEL} versions:
+
+* 9.0 or later
+* 8.0 or later
+* 7.0 or later
+* 6.1 or later*
++
+NOTE: Red{nbsp}Hat Enterprise{nbsp}Linux versions 6.1, 6.2, and 6.3 require `subscription-manager` and related packages to be updated manually.
```

Comment:
This is almost the RHEL macro, but `{RHEL}` doesn't use `{nbsp}` in Enterprise Linux. Perhaps it should?

Reply:
I plan to update supported OS versions in following PRs.